include/cantera/kinetics/InterfaceRate.h

```diff
@@ -243,7 +254,11 @@ class InterfaceRateBase
     map<size_t, size_t> m_indices;
     vector<string> m_cov; //!< Vector holding names of coverage species
     vector_fp m_ac; //!< Vector holding coverage-specific exponential dependence
-    vector_fp m_ec; //!< Vector holding coverage-specific activation energy dependence
+    //! Vector holding coverage-specific activation energy dependence as a
+    //! 5-membered array of polynomial coeffcients starting from 0th-order to
+    //! 4th-order coefficients
+    std::vector<vector_fp> m_ec;
+    std::vector<bool> m_lindep; //!< Vector holding boolean for linear dependence
```

Comment:
That's great to know. While I am at it, shall I remove `std::` for the other parts of the `InterfaceRate` where applicable? Or change only for what I added?

Reply:
The only necessary changes are for lines of code that you added/modified. If you want, you can update nearby lines in the same function, but please do not expand the scope beyond that, since it makes for large diffs that obscure the non-trivial code changes.

Reply:
Looking at the commit you made to make some of these changes: this is fine -- there aren't that many of them in this particular file, so I think it was OK to update them.